.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies and run the release task
+
+name: Release
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+*"
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby-version: ['3.2']
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Ruby
+    # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+    # change this to (see https://github.com/ruby/setup-ruby#versioning):
+      uses: ruby/setup-ruby@55283cc23133118229fd3f97f9336ee23a179fcf # v1.146.0
```

Comment:
I need to learn to read! I see the line I thought was a duplicate actually "belongs with" the previous two comment lines. Now my question is: Is there a reason to use this specific version vs. the more generic option? Is the specific version just to be on the safe side?

Reply:
The stuff for this part was actually auto-generated when I initially created the workflow (there was a way to do that in GitHub to create a template or something), so I'm not quite sure why it created it with the sha hash instead of the recommended `v1`.  I will give it a test on the clone branch just to make sure first.

Reply:
Changing this to `@v1` on the clone repo worked fine, so I'll make the change here and some minor changes to the comment as well.